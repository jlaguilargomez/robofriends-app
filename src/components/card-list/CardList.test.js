import React from 'react';
import { shallow } from 'enzyme';
import CardList from './CardList';

describe('CardListComponent', () => {
  const mockRobots = [
    {
      id: 1,
      name: 'Se√±or Pepe',
      username: 'SrPepe',
      email: 'srpepe@email.com',
    },
  ];
  const app = shallow(<CardList robots={mockRobots}></CardList>);


  it('expect to render CardList component', () => {
    
    // 'shallow' nos permite testear componentes sin depender de sus sub-componentes (poco-profundo)
    expect(app).toMatchSnapshot();
  });
})
