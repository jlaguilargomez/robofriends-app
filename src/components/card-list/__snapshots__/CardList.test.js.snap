// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardListComponent expect to render CardList component 1`] = `
<div>
  <Card
    email="srpepe@email.com"
    id={1}
    key="1"
    name="SeÃ±or Pepe"
  />
</div>
`;
